democracy/enums.py

```diff
@@ -14,9 +14,8 @@ class Labels:
         OPEN = _("Open commenting")
 
 
-class SectionType(Enum):
+class InitialSectionType:
```

Comment:
I wonder if `SpecialSectionType` or something might be a better name for this...